resources/config_skeleton.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
maybe `config.json.skeleton` is easier to change?

Reply:
@ahalberkamp this is not addressed?

Reply:
af85665